Smooch.postman_collection.json

```diff
@@ -10632,6 +10632,180 @@
 						}
 					]
+				},
+				{
+					"name": "Location Message",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "Authorization",
+								"value": "Bearer {{JWT}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"role\": \"appMaker\",\n    \"type\": \"location\",\n    \"coordinates\": {\n        \"lat\": 45.5261583,\n        \"long\": -73.595346\n    }\n,   \"location\": {\n        \"address\": \"5333 avenue Casgrain\",\n        \"name\": \"Zendesk Montreal\"\n    }\n}"
+						},
+						"url": {
+							"raw": "{{url}}/{{apiVersion}}/apps/{{appId}}/appusers/{{appUserId}}/messages",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"{{apiVersion}}",
+								"apps",
+								"{{appId}}",
+								"appusers",
+								"{{appUserId}}",
+								"messages"
+							]
+						},
+						"description": "### For more information\n- Guide: [https://docs.smooch.io/guide/sending-messages/](https://docs.smooch.io/guide/sending-messages/)\n- API Docs: [https://docs.smooch.io/rest/#location](https://docs.smooch.io/rest/#location)\n\nSee the [capabilities grid](https://docs.smooch.io/guide/channel-capabilities/) for message-type support across channels."
+					},
+					"response": [
+						{
+							"name": "Location Example",
+							"originalRequest": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"role\": \"appMaker\",\n    \"type\": \"location\",\n    \"coordinates\": {\n        \"lat\": 45.5261583,\n        \"long\": -73.595346\n    }\n,   \"location\": {\n        \"address\": \"5333 avenue Casgrain\",\n        \"name\": \"Zendesk Montreal\"\n    }\n}"
```

Comment:
nit:

```suggestion
									"raw": "{\n    \"role\": \"appMaker\",\n    \"type\": \"location\",\n    \"coordinates\": {\n        \"lat\": 45.5261583,\n        \"long\": -73.595346\n    }\n,   \"location\": {\n        \"address\": \"5333 avenue Casgrain\",\n        \"name\": \"Zendesk Montréal\"\n    }\n}"
```

<img width="25" alt="parrot" src="https://cultofthepartyparrot.com/parrots/hd/parrot.gif" />

(reason for my nits being: I believe the `é` is part of Zendesk's official branding for the Montréal office)